chatgpt/utils/proof_of_contribution.py

```diff
@@ -79,7 +79,7 @@ def download_and_decrypt_file(input_url, input_encryption_key):
         return decrypted_file_path
 
 
-def validate_file(decrypted_file_path):
+def proof_of_quality(decrypted_file_path):
```

Comment:
It's not really part of your PR, but I think we should additionally refactor so that chatgpt.utils.validator#validate_chatgpt_zip is more clearly a quality check. Maybe that would be as simple as renaming it to `score_quality`.

Reply:
There's some metadata calculations in there too so I renamed to `evaluate_chatgpt_zip`